### PR TITLE
Fix all lint warnings

### PR DESCRIPTION
**Pull request by SaraVieira**
It's impossible to see when there is a lint error since we had like 70 warnings in the whole app

This fixed all the warnings

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/5550/commits/378fab5a535ca99b1da9f61e87cf6bf66751d834 is at https://pr5550.build.csb.dev/s/new.

File: packages/app/src/sandbox/compile.ts [hunk 1]
```diff
@@ -657,14 +658,16 @@ async function compile({
...
-        const { head, body } = getHTMLParts(
-          htmlModule && htmlModule.code
-            ? htmlModule.code
-            : template === 'vue-cli'
-            ? '<div id="app"></div>'
-            : '<div id="root"></div>'
-        );
+        let html = '<div id="root"></div>';
+        if (htmlModule && htmlModule.code) {
+          if (htmlModule.code) {
```
Comment (CompuIves) [inline packages/app/src/sandbox/compile.ts:663]: I think here there should be html = htmlModule.code and in the if statement the Vue check.

**Review by CompuIves — CHANGES_REQUESTED** [review]
Wooow, very impressive. This must've been a ton of work! I checked all files and I only found one thing where it mightve converted wrongly. Left a comment there.

Would be good to merge and deploy tomorrow, then we can take the day to monitor sentry to be sure that everything still works!

File: packages/app/src/sandbox/compile.ts (hunk 1 above)
Comment (christianalfoni) [reply packages/app/src/sandbox/compile.ts:663]: Ah, yeah, so:

```ts
let html = template === 'vue-cli' ? '<div id="app"></div>' : '<div id="root"></div>';
if (htmlModule && htmlModule.code) {
  html = htmlModule.code
}
```

**Review by christianalfoni — CHANGES_REQUESTED** [review]
This is a SOLID piece of work, thanks so much for doing this @SaraVieira !! Just one tiny logical thing that needs to be fixed 😄

File: packages/app/src/sandbox/compile.ts (hunk 1 above)
Comment (SaraVieira) [reply packages/app/src/sandbox/compile.ts]: truuu, just pushed

**Review by christianalfoni — APPROVED** [review]
Great! 👍  😄

**Pull request merged**